dev/dev.html

```diff
@@ -75,9 +75,12 @@
         <v-select placeholder="filtering with fuse.js" label="title" :options="fuseSearchOptions" :filter="fuseSearch">
             <template slot="option" scope="option">
                 <strong>{{ option.title }}</strong><br>
-                <em>{{ `${option.author.firstName} ${option.author.lastName}` }}</em>
+                <em>{{ option.author.firstName + ' ' + option.author.lastName }}</em>
```

Comment:
is there a reason to not use interpolation here? The prior code style was probably the way to go

Reply:
This file was was never compiled and could therefore not be run in internet explorer. I would leave it like this to make it testable in IE.